.eslintrc

```diff
@@ -61,9 +61,6 @@
 		"@typescript-eslint/prefer-for-of": "error",
 		"@typescript-eslint/prefer-includes": "error",
 		"@typescript-eslint/prefer-nullish-coalescing": "error",
-		"@typescript-eslint/prefer-string-starts-ends-with": "error",
-		"@typescript-eslint/consistent-type-imports": ["error", {
```

Comment:
I liked this rule because it told to ts compiler that some things are only used as types and not values, so it could be more performant in compile process

well I guess we have to live with it

Reply:
Well, this rule is really good, but often we have a situation where we have to import something like this
```ts
import { Foo } from './foo';
import type { Bar } from './foo';
```
And then the linter warns that we have duplicate imports.(

Reply:
this was actually the rule that I was thinking of that supports type imports also
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-duplicates.md
we use it at work, for the same reason because it is smarter than the one in eslint core

Reply:
You can try to implement.)